/**:
  ros__parameters:

    x: -0.8183830985020199
    y: -3.7414036212776987
    z: 0.6499999999999999
    yaw: 1.612959581471831

    goal_x: 1.46
    goal_y: 3.58
    goal_z: 0.35